tracee-ebpf/main.go

```diff
@@ -1158,6 +1182,45 @@ func unpackBPFBundle(dir string) error {
 			return fmt.Errorf("error copying bpf file: %s", err.Error())
 		}
 	}
+
+	return nil
+}
+
+// unpackBTFHub unpacks tailored, to the compiled eBPF object, BTF files for kernel supported by BTFHub
+func unpackBTFHub(outFilePath string, OSInfo *helpers.OSInfo) error {
+	if !btfEmbedded {
+		return fmt.Errorf("there are no embedded BTF files available")
+	}
+
+	var btfFilePath string
+
+	osId := OSInfo.GetOSReleaseFieldValue(helpers.OS_ID)
```

Comment:
It might be a problem to use this API of OSInfo in tracee container (alpine based) as we will get the wrong distro release info (aquasecurity/libbpfgo#92)

Reply:
Yes, we're mitigating this by doing: https://github.com/aquasecurity/tracee/pull/1227/commits/76883662956efb30e9d7d22b00d9a5a719492624#diff-9622619b5207ed1b3ace8ccab271f0f000e5271e1d6412a5b598a3171f5f1080R47